src/database/migrations/20240410100649-add-verified-to-user.js

```diff
@@ -0,0 +1,14 @@
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.addColumn('users', 'verifiedAccount', {
```

Comment:
we already have this verified use it

Reply:
The verified we have is the Google verified, so we need another verified that verifies account on our platform (excluding google).

That is why we use another field **verifiedAccount**

Reply:
please remove this it's was already added by nadine